integrationtests/run_tests.bash

```diff
@@ -31,16 +33,61 @@ while getopts "h?og" opt; do
         ;;
     g)  gui=true
         ;;
+    v)  verbose=true
+		;;
     esac
 done
 
+if $verbose 
+	then
+	echo "run_tests.bash called."
+fi
+
 
 # determine the directory of the source given the directory of this script
 pushd `dirname $0` > /dev/null
 SCRIPTPATH=`pwd`
 popd > /dev/null
 ORIG_SRC=$(dirname $SCRIPTPATH)
 
+# =============================================================================================== #
+# =================================== Install/build OpticalFlow ================================= #
+# =============================================================================================== #
+echo "=====> Building/installing OpticalFlow"
```

Comment:
I don't like the idea of adding all of these submodules. See general comment on thread.